src/ontodia/widgets/node.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { FatClassModel } from '../diagram/elements';
+import { formatLocalizedLabel } from '../diagram/model';
+import { TreeNodes } from './treeNodes';
+
+export interface NodeTreeProps {
+    node: FatClassModel;
+    lang?: Readonly<string> | undefined;
+    searchResult?: Array<FatClassModel> | undefined;
+    searchString?: string | undefined;
+    onClassSelected: (classId: string) => void;
+    onDragDrop?: (e: DragEvent, paperPosition: { x: number; y: number; }) => void;
+}
+
+export interface ClassTreeState {
+    expanded?: Boolean | undefined;
+    bgColor?: string | undefined;
```

Comment:
It's better to use style rules for class names instead of inline styles.

Reply:
I understand, but I did it so compulsorily.
The matter is that on a clique on a class in a tree I process search of elements in Instances. If in styles to set properties at a clique, they will be redefined inline styles to search for clicks on a click.

Reply:
`onClick={this.showInstances}` simultaneously selects the background of the class and looks for instances.